tests/NuGetGallery.FunctionalTests.Core/EnvironmentSettings.cs

```
@@ -18,43 +18,28 @@ public class EnvironmentSettings
         private static string _testAccountPassword;
         private static string _testAccountApiKey;
         private static string _testEmailServerHost;
-        private static string _runFunctionalTests;
-        private static string _readOnlyMode;
+        private static bool? _runFunctionalTests;
         private static List<string> _trustedHttpsCertificates;
 
         /// <summary>
         /// Option to enable or disable functional tests from the current run.
         /// </summary>
-        public static string RunFunctionalTests
+        public static bool RunFunctionalTests
         {
             get
             {
-               if (string.IsNullOrEmpty(_runFunctionalTests))
+                if (!_runFunctionalTests.HasValue)
                 {
-                    if (string.IsNullOrEmpty(Environment.GetEnvironmentVariable("RunFunctionalTests")))
-                        _runFunctionalTests = "False";
-                    else
-                        _runFunctionalTests = Environment.GetEnvironmentVariable("RunFunctionalTests");
-                }
-                return _runFunctionalTests;
-            }
-        }
+                    bool runFunctionalTests;
+                    if (!bool.TryParse(Environment.GetEnvironmentVariable("RunFunctionalTests"), out runFunctionalTests))
```

1. if the config is empty, the user didn't add a value to the setting, and we should fallback to default behavior (what ever we decide it is). 
If the user wrote "blablabla" in the setting, we need to fail on parsing, and show that we failed on parsing, instead of swallowing the error.